backend/services/routes/homepage.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const router = express.Router();
+
+const axios = require("axios");
+axios.defaults.baseURL = "http://localhost:3000";
+
+function getProducts() {
+  return axios.get("/api/products");
```

Comment:
Why do we need this methods here in the router?
Routes should use internal core-layer methods instead of creating +1 more call to endpoint. It's a wrong approach, please fix it.

Recheck this point for all routes